.github/workflows/lockfile.yml

```diff
@@ -1,8 +1,12 @@
 name: Check Lockfile
-on: pull_request
+on:
+  pull_request:
+    paths:
+      - 'package-lock.json'
```

Comment:
highlighting this since it's kind of out of scope for the PR and could be reverted if not desired. Figured it would be worth doing to save a few CPU cycles though 😄

Reply:
I'm fine with including this.